packages/ts-codegen/src/__tests__/context.test.ts

```diff
@@ -2,10 +2,81 @@ import ts from "typescript";
 import { parse } from "graphql";
 import { blankGraphQLTag as graphql } from "../utilities";
 import { generateTS } from "..";
-import { ContextMap } from "../context";
+import { SubTypeNamespace } from "../codegen";
+import { OutputMetadata } from "../context/utilities";
 
 describe(generateTS, () => {
   describe("Tests basic syntax GraphQL syntax", () => {
+    const contextSubTypeMetadata = {
+      managers: {
+        user: {
+          name: "user",
+          importTypeName: 'UserStateMachineType["user"]',
+          importPath: "@package/user-state-machine",
```

Comment:
is it actually a `path`? can I use relative paths here?

Reply:
yes, it can be either package or relative path